pkg/meroxa/connector.go

```diff
@@ -59,6 +60,7 @@ type Connector struct {
 type CreateConnectorInput struct {
 	Name          string                 `json:"name,omitempty"`
 	ResourceName  string                 `json:"resource_name"`
+	Application   ApplicationIdentifier  `json:"application"`
```

Comment:
```suggestion
	Application   ApplicationIdentifier  `json:"application"`
```
the view has not been updated